modules/lvms-about-lvmcluster-cr.adoc

```diff
@@ -130,6 +130,24 @@ For example, if this field is set to 10, you can provision up to 10 times the am
 
 To disable over-provisioning, set this field to 1.
 
+|`thinPoolConfig.chunkSize`
+|`integer`
+|Specifies the statically calculated chunk size for the thin pool. It is used only when the `ChunkSizeCalculationPolicy` field is set to `Static`. The value for this field must be configured in the range 64 KiB to 1 GiB due to the underlying limitations of `lvm2`. 
+
+If this field is not configured, and the `ChunkSizeCalculationPolicy` field is set to `Static`, the default chunk size is `128KiB`.
```

Comment:
```suggestion
If this field is not configured and the `ChunkSizeCalculationPolicy` field is set to `Static`, the default chunk size is set to `128KiB`.
```

Reply:
You could remove the passive voice by stating "you" and this moves any ambigousy away. Unless some OCP component sets the value for the field and not a product user? This would align with the style voice for the next field.

Reply:
Updated as follows:
"If you do not configure this field and the `ChunkSizeCalculationPolicy` field is set to `Static`, the default chunk size is set to 128 KiB."